### DISCUSSION TITLE
Prettier Plugins

### DISCUSSION THREAD

**Discussion by fweth** [codesandbox/codesandbox-client#6595] [Q&A] [answered]
How do Prettier plugins work in CodeSandbox?  I just tried out Svelte, there seems to be some formatting happening, at least inside the script tag.  But that's weird, because vanilla Prettier can't deal with Svelte files at all, as far as I can see?  Also, the `prettier-plugin-svelte` is supposed to format all parts of Svelte code, yet installing it in CodeSandbox seems to make no difference.

**Comment by JamesACS** [comment, marked as answer]
@fweth - Did you manage to resolve this?

**Reply by fweth** [reply]
Not really, does CodeSandbox support Prettier plugins in theory?  What would be the installation process, just adding the NPM package or something else?

**Reply by JamesACS** [reply]
We do already have a Prettier integration, you can change the settings from the preferences window!

**Reply by fweth** [reply]
Hey, I was not referring to the .prettierrc.json, I wanted to install an npm package which extends Prettier, because apperently Prettier cannot be configured to parse Svelte files.  But since I'm not really using Svelte, just wanted to play around with it, I'm going to close this thread.  Thanks for the replies :)
